test/Libraries/Microsoft.Extensions.Diagnostics.Probes.Tests/TcpEndpointHealthCheckServiceTests.cs

```diff
@@ -7,7 +7,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging.Testing;
-using Microsoft.Extensions.Time.Testing;
+using Microsoft.Extensions.TimeProvider.Testing;
```

Comment:
The issue with this namespace that it will clash with the actual type name
![image](https://github.com/dotnet/extensions/assets/4403806/f207abba-9a7f-4a62-a576-44684f7a6635)